amethyst_tiles/src/map.rs

```diff
@@ -20,6 +20,11 @@ pub trait Tile: 'static + Clone + Send + Sync + Default {
     fn tint(&self, coordinates: Point3<u32>, world: &World) -> Srgba {
         Srgba::new(1.0, 1.0, 1.0, 1.0)
     }
+
+    /// Takes an immutable reference to world to process this sprite and return its flip.
+    fn flip(&self, coordinates: Point3<u32>, world: &World) -> (bool, bool) {
```

Comment:
I'm not good at naming stuff. I'm hoping a better suggestion can be made here for the function name.

Reply:
The name is fine, but could you add documentation to this function to specify what the tuple return value should be? It should explicit state that it needs to return (horizontal, vertical)